src/cow-react/modules/limitOrders/pure/ReceiptModal/FeeField.tsx

```diff
@@ -8,12 +8,12 @@ export type Props = { order: ParsedOrder }
 export function FeeField({ order }: Props): JSX.Element | null {
   const { executedFeeAmount, inputToken, sellToken } = order
 
-  let formattedExecutedFee: string | undefined = executedFeeAmount?.toString(10) || ''
+  let formattedExecutedFee: string | undefined = executedFeeAmount
   let quoteSymbol: string | undefined = sellToken
   let executedFeeCurrency: CurrencyAmount<Token> | undefined
 
   if (sellToken) {
-    executedFeeCurrency = CurrencyAmount.fromRawAmount(inputToken, executedFeeAmount?.toString() || 0)
+    executedFeeCurrency = CurrencyAmount.fromRawAmount(inputToken, executedFeeAmount || 0)
```

Comment:
Please, use `tryParseCurrencyAmount` instead

Reply:
It doesn't give me the correct result with `tryParseCurrencyAmount`. It gives me 
![Screenshot from 2022-12-01 14-22-51](https://user-images.githubusercontent.com/34926005/205063958-c0bc1a33-cf42-4273-98bd-e62c3af0e4d4.png)
when it should be 
![Screenshot from 2022-12-01 14-22-41](https://user-images.githubusercontent.com/34926005/205063978-94288540-b57a-49b8-925a-f908393c65e9.png)

Reply:
This is the order in question https://protocol-explorer.dev.gnosisdev.com/goerli/orders/0x2f4c4c4d9e6ff75fe588ee4acce57dcbb8289ffaf14068db0d70f04c4e63a94ef17afe5237d982868b8a97424dd79a4a50c364126388ad6b

Reply:
But just doing `CurrencyAmount.fromRawAmount(inputToken, executedFeeAmount || 0)` fixes the issue, because there is no need to convert that value to BigNumber

Reply:
Hmmm, then ok